src/composables/accountAssetsList.ts

```diff
@@ -50,12 +51,18 @@ export function useAccountAssetsList({
     return currentBalance || new BigNumber(0);
   });
 
+  const accountProtocol = computed(
+    () => isMultisig ? PROTOCOLS.aeternity : activeAccount.value.protocol,
+  );
```

Comment:
I suggest to create a computed property that would return `activeAccount` or `multisigActiveAccount` based on the `isMultisig` property. Then take the `protocol` from this computed property instead.

This would allow to update lines 59-61 with this computed property.

Reply:
`IMultisigAccount` type doesn't have `procotol` inside

Reply:
My bad. Skip my comment on this.